src/App.tsx

```diff
@@ -71,7 +112,63 @@ const App = observer(() => {
   if (!store.isStarting) {
     return (
       <Layout>
-        <Header />
+        <Header handleOpenChat={handleOpenChat} />
+        {openChat.shouldOpenMaximizedChat === true && (
+          <ChatBox
+            recipientEmail={openChat.recipientEmail!}
+            profilePicture={
+              openChat.profilePicture !== undefined
+                ? openChat.profilePicture
+                : null
+            }
+            onMinimize={() => {
+              setOpenChat({
+                ...openChat,
+                shouldOpenMaximizedChat: false,
+                shouldOpenMinimizedChat: true,
+              });
+            }}
+            onClose={() =>
+              setOpenChat({
+                shouldOpenMaximizedChat: false,
+                shouldOpenMinimizedChat: false,
+                recipientEmail: undefined,
+                profilePicture: undefined,
+                name: undefined,
+                surname: undefined,
+                profile: undefined,
+              })
+            }
+            name={openChat.name ?? ""}
+            surname={openChat.surname ?? ""}
+            profile={t(openChat.profile ?? "")}
+          />
+        )}
+        {openChat.shouldOpenMinimizedChat === true && (
```

Comment:
Podobnie jak wyżej:  
`openChat.shouldOpenMinimizedChat` to boolean, nie trzeba tutaj porównywać z `true`, wystarczy samo: 
```suggestion
        {openChat.shouldOpenMaximizedChat && (
```